backbone/src/main/res/drawable/rsb_tremortest1.xml

```diff
@@ -0,0 +1,88 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Yeah, I can totally see that.  I was just copying iOS.  I'll rename them to something more like you suggested.